scripts/generate_docs.sh

```diff
@@ -1,5 +1,25 @@
 #!/usr/bin/bash
 
+RELEASE=""
```

Comment:
Could you describe how to use this script with a new options in the heading? I mean e.g.:
~~~bash
# ./generate_docs.sh --increment # Increment "z" in version
# ./generate_docs.sh --keep # Keep current version
~~~

Reply:
added